torch/special/__init__.py

```diff
@@ -415,6 +417,74 @@
     tensor([2.1972, 1.3863, 0.0000])
 """.format(**common_args))
 
+j0 = _add_docstr(_special.special_j0,
+                 r"""
+j0(input, *, out=None) -> Tensor
+
+Computes the zero order Bessel function of the first kind for each element of :attr:`input`.
+
+Example::
+    >>> torch.special.j0(torch.arange(5, dtype=torch.float32))
+    tensor([1.0000, 0.4658, 0.3085, 0.2430, 0.2070])
```

Comment:
The examples should go after the args and kwargs. Same applies to all the others.

Reply:
Oops! Fixed.